akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -528,7 +528,7 @@ object CompactByteString {
     else {
       val ar = new Array[Byte](bytes.size)
       bytes.copyToArray(ar)
-      CompactByteString(ar)
+      ByteString.ByteString1C(ar)
```

Comment:
ah, good point, I must have looked at wrong apply

is that copyToArray more efficient than `bytes.toArray.clone`?